adapters/Vungle/VungleAdapter/VungleRouter.m

```diff
@@ -243,25 +278,37 @@ - (UIView *)renderBannerAdInView:(UIView *)bannerView
                                             placementID:placementID
                                                   error:&bannerError];
   if (success) {
-    self.isMrecPlaying = YES;
+    if (delegate.adapterAdType == MREC) {
+        self.isMrecPlaying = YES;
+    } else {
+        self.isBannerPlaying = YES;
+    }
 
     return bannerView;
   } else {
     NSLog(@"Banner loading error: %@", bannerError.localizedDescription);
-    self.isMrecPlaying = NO;
-
-    return nil;
+    if (delegate.adapterAdType == MREC) {
+        self.isMrecPlaying = NO;
+    } else {
+        self.isBannerPlaying = NO;
+    }
   }
 
   return nil;
 }
 
 - (void)completeBannerAdViewForPlacementID:(NSString *)placementID {
-  if (placementID && self.isMrecPlaying) {
+  if (placementID && (self.isMrecPlaying || self.isBannerPlaying)) {
     NSLog(@"Vungle: Triggering an ad completion call for %@", placementID);
-    self.isMrecPlaying = NO;
-
+      
     [[VungleSDK sharedSDK] finishedDisplayingAd];
+    if (self.isMrecPlaying) {
+        self.isMrecPlaying = NO;
+        self.mrecPlacementID = nil;
```

Comment:
Set banner/mrec placement to nil after finishing playing banner/mrec ad, so adapter can load next banner/mrec with different placement ID.

Reply:
@YueVungle Looks good. I will keep my thumbs up, but please verify "refresh" works correctly after your update here.

Reply:
@ashinagawa Hi Aki, I have verified that "refresh" works correctly after the update here. :)
It seems like this PR has passed the code review, may I close this RR :)